annotation.html

```diff
@@ -0,0 +1,125 @@
+<!doctype html>
+<html>
+    <head>
+        <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+        <meta content="utf-8" http-equiv="encoding">
+        <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+        <link rel="stylesheet" type="text/css" href="css/annotation.css">
+        <link rel="stylesheet" type="text/css" href="css/font-awesome.min.css">
+        <title>
+            Annotation of an URL if any. 
+        </title>
+    </head>
+    <body>
+        <table width="100%" style="max-height: 200px; ">
+            <tr class="spaceUnder">
+                <td style="text-align: center;">
+                    <div id ="horizontal_line" style="margin-bottom: 5px"></div>
+                        <div class='container' style="background: white">
+                            <img class='resize_fit_center'
+                            src='images/logo.gif' />
+                        </div>
+                    <div id="horizontal_line" style="margin-top:3px"></div>
+                </td>
+            </tr>
+        </table>
+        <div>
+            <h2 style="text-align: center;"><strong>Annotations of Current Domain</strong><br><span style="font-size: 50%;font-family:sans-serif"> From <a style="cursor: pointer;" href="http://hypothes.is/">Hypothes.is</a></span></h2>
+        </div>
+        <div class="container" id="container-whole">
+
+        </div>
+        <div class="row" id="row-contain" style="    display: flex;
+        align-items: stretch;
+        justify-content: center;" >
+            <div class="col-sm-6" style="width: 600px">
+                <div class="box">
+                    <div class="top-div" id="top-div-contain">
+                        <strong>
+                            <span id="userinfo">
+
+                            </span>
```

Comment:
Why don't you trim empty lines and make this `<span id="userinfo"></span>`. Please follow a consistent style everywhere and try to make the HTML more compact when possible (without hurting readability).

Reply:
Thank you . I will do that . I will devote some time to make these changes to increase readability.